packages/create-widget.template.react.v2/templates/package.json.hbs

```diff
@@ -7,7 +7,7 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint . --report-unused-disable-directives --max-warnings 0",
-    "test": "vitest run",
+    "test": "vitest run --passWithNoTests",
```

Comment:
Why did we add this?

Reply:
Because `pnpm check` fails on the widget example if you don;t have it. not sure how this passed on main. 
```
@osdk/example-generator:test: cache hit, suppressing logs 70c9d4e43fb9fad6
@osdk/examples.widget-react-sdk-2.x:test: 
@osdk/examples.widget-react-sdk-2.x:test: > @osdk/examples.widget-react-sdk-2.x@0.0.0 test /Volumes/git/osdk-ts/examples/example-widget-react-sdk-2.x
@osdk/examples.widget-react-sdk-2.x:test: > vitest run
@osdk/examples.widget-react-sdk-2.x:test: 
@osdk/examples.widget-react-sdk-2.x:test: 
@osdk/examples.widget-react-sdk-2.x:test:  RUN  v2.1.3 /Volumes/git/osdk-ts/examples/example-widget-react-sdk-2.x
@osdk/examples.widget-react-sdk-2.x:test: 
@osdk/examples.widget-react-sdk-2.x:test: include: **/*.{test,spec}.?(c|m)[jt]s?(x)
@osdk/examples.widget-react-sdk-2.x:test: exclude:  **/node_modules/**, **/dist/**, **/cypress/**, **/.{idea,git,cache,output,temp}/**, **/{karma,rollup,webpack,vite,vitest,jest,ava,babel,nyc,cypress,tsup,build,eslint,prettier}.config.*
@osdk/examples.widget-react-sdk-2.x:test: 
@osdk/examples.widget-react-sdk-2.x:test: No test files found, exiting with code 1
@osdk/examples.widget-react-sdk-2.x:test:  ELIFECYCLE  Test failed. See above for more details.
@osdk/examples.widget-react-sdk-2.x:test: ERROR: command finished with error: command (/Volumes/git/osdk-ts/examples/example-widget-react-sdk-2.x) /Users/ewitkon/.nvm/versions/node/v18.19.0/bin/pnpm run test exited (1)
@osdk/examples.widget-react-sdk-2.x#test: command (/Volumes/git/osdk-ts/examples/example-widget-react-sdk-2.x) /Users/ewitkon/.nvm/versions/node/v18.19.0/bin/pnpm run test exited (1)
```